doc/data/messages/t/too-complex/good.py

```diff
@@ -1 +1,39 @@
-# This is a placeholder for correct code for this message.
+# split up to smaller functions
+
+fruit = {'apple': 1.1,
+         'pear': 0.8,
+         'banana': 1.2,
+         'mango': 3.5,
+         'peach': 0.5,
+         'melon': 4.9,
+         'orange': 2.0,
+         'strawberry': 2.5,
+         'mandarin': 2.3,
+         'plum': 0.5,
+         'watermelon': 6.4
+         }
+
+
+def fifty_percent_off(whole):
+    return (float(whole)) * 50 / 100
+
+
+def total_sum(list1):
+    total = 0
+    for i in list1:
+        total += i
+    return total
+
+
```

Comment:
```suggestion
```

Reply:
> Looks pretty nice already, thank you for creating an elaborate example ! Could you apply black on it too, it's simply to have a better formatting overall :)

Thank you! Can you elaborate on applying black to it. How would i change the formatting on the page?

Reply:
black is an autoformatter, you can install it with ``pip install black``, then you can apply it on good.py and bad.py to format them automatically.